### ISSUE TITLE
I loose webuiPopover after swap position

### ISSUE THREAD

**Issue by xmon** [sandywalker/webui-popover#135]
Hi,

I have a problem when I use [zepto-dragswap](https://github.com/james2doyle/zepto-dragswap)

After swap elements I loose the webuiPopover.

I solve redefining after "dropComplete", but I duplicate code.

any idea to reactivate webuiPopover?

thanks a lot

**Comment by sandywalker** [comment]
Maybe there's event conflicted, You can set set the option  'dismissible' to false and try again.

**Comment by xmon** [comment]
Hi, 

thanks for fast reply.

I think maybe I did not explained well.

anyway, I tried your suggestion, and it continues to behave the same, except that the webuipopover to click outside it does not close.

The real problem is that when I swap positions with the other plugin, [zepto-dragswap](https://github.com/james2doyle/zepto-dragswap), what it does, is dynamically modify DOM elements, including the link behavior is assigned webuipopover show. Performing this action, the allocation of webuipopover show is lost, and so far the only solution I've found, is to re-initialize the event after confirming the exchange of DOM elements "dropComplete".

I would find a solution, because the code included in the implementation of webuipopover is quite long, and I show a form that is sent by Ajax. control 3 or 4 steps until the server response is received and action on a DOM element are taken, such as updates on the fly.

thanks again
